apps/dialtone-documentation/docs/guides/content/grammar-and-mechanics.md

```diff
@@ -0,0 +1,1266 @@
+---
+title: Grammar and mechanics
+description: These guidelines exist to help everyone at Dialpad write helpful copy that shares the same voice, tone, and consistency.
+---
+
+## Active voice
+
+We help users accomplish tasks every day. To reflect that acitvity, we write using an active voice in our copy. Use this voice for when the user needs to do something. We only allow a passive voice in very specific scenarios. Otherwise, use an active voice in all copy you write.
+
+To write in an active voice, you place the subject at the start of a sentence. It clearly identifies who is performing the action. This results in more direct and concise writing. It should be clear that the subject is the one doing the action.
+
+> Subject (Person / Thing) + Verb (the action) + Object (What receives the action)
+
+<section class="d-d-grid d-gg16 d-g-cols2">
+  <dt-card class="d-jc-flex-start d-bs-none d-ba d-bc-subtle d-bar12" headerClass="d-p0">
+    <template #header>
+      <div class="d-d-flex d-ai-center d-mn1 d-gg4 d-px16 d-py6 d-btr12 d-ba d-bc-green-400 d-bco25 d-bgc-success d-fc-success d-fs-100" style="--bco: .25; width: calc(100% + 2px)"><dt-icon name="check" size="200" /> <strong>Do</strong></div>
+    </template>
+    <template #content>
+      <p>Calls will be sent to voicemail.</p>
+    </template>
+  </dt-card>
+  <dt-card class="d-jc-flex-start d-bs-none d-ba d-bc-subtle d-bar12" headerClass="d-p0">
+    <template #header>
+      <div class="d-d-flex d-ai-center d-mn1 d-gg4 d-px16 d-py6 d-btr12 d-ba d-bc-red-400 d-bco25 d-bgc-critical d-fc-critical d-fs-100" style="--bco: .15; width: calc(100% + 2px)"><dt-icon name="close" size="200" /> <strong>Don't</strong></div>
+    </template>
+    <template #content>
+      <p>Missed calls were sent to voicemail.</p>
+    </template>
+  </dt-card>
+</section>
```

Comment:
We already have a VuePress doc component for "Do/Don't".

https://github.com/dialpad/dialtone/blob/staging/apps/dialtone-documentation/docs/components/button.md?plain=1#L48-L61

Reply:
Ok. I'll update them. I'll probably update the Do/Don't component as well.